tests/bcr/go_mod/MODULE.bazel

```diff
@@ -14,9 +14,12 @@ local_path_override(
     path = "test_dep",
 )
 
+bazel_dep(name = "bazel_features", version = "1.14.0")
 bazel_dep(name = "protobuf", version = "23.1", repo_name = "my_protobuf")
 bazel_dep(name = "rules_go", version = "0.42.0", repo_name = "my_rules_go")
+bazel_dep(name = "rules_license", version = "0.0.8")
 bazel_dep(name = "rules_proto", version = "6.0.0-rc2", repo_name = "my_rules_proto")
+bazel_dep(name = "rules_testing", version = "0.6.0")
```

Comment:
Isn't this a dev dependency?

Reply:
It's a regular dependency of the test module. Since nothing depends on the test module, we don't usually declare deps as dev-only.